packages/frontend/src/redux/actions/account.js

```diff
@@ -625,12 +660,12 @@ export const getAvailableAccountsBalance = () => async (dispatch, getState) => {
 
     for (let i = 0; i < availableAccounts.length; i++) {
         const accountId = availableAccounts[i];
-        if (!accountsBalance || !accountsBalance[accountId]) {
+        if (!accountsBalance[accountId]) {
             i < 0 && await dispatch(setAccountBalance(accountId));
```

Comment:
Irrelevant to your changes here but I'm curious 🙂  - can this ever be executed if `i` starts at 0 and only ever gets incremented?

Reply:
:) Correct, this was a part of the functionality that I prepared some time ago - we wanted to show balances for several first accounts when the user opens submenu (and has more than one account) - after that, we were worried, that it might affect efficiency, because balance calculation contained lockup and staking, and the fetch was heavy, so we disabled fetching it automatically, but allows user to fetch it manually until we figure out how to make it work faster. So that's why we have `i < 0` :) 

Now by default balances in the submenu are hidden, so I guess we don't need this functionality at all. @stefano are we planning to get back to showing balances by default in the submenu in near future?

Anyway, not a blocker for this PR, I will remove it after Stefano's confirmation in a separate PR.

Reply:
Thanks for the context!